src/sentry/sentry_metrics/indexer/base.py

```diff
@@ -449,23 +482,25 @@ def record(self, use_case_id: UseCaseID, org_id: int, string: str) -> Optional[i
         """
         raise NotImplementedError()
 
-    def resolve(self, use_case_id: UseCaseKey, org_id: int, string: str) -> Optional[int]:
+    @metric_path_key_compatible_resolve
+    def resolve(self, use_case_id: UseCaseID, org_id: int, string: str) -> Optional[int]:
         """Lookup the integer ID for a string.
 
         Does not affect the lifetime of the entry.
 
-        Callers should not rely on the default use_case_id -- it exists only
-        as a temporary workaround.
+        This function is backwards compatible with UseCaseKey while call sites that still uses
```

Comment:
tiny nit: this comment is only present for the Postgres indexer's resolve and reverse_resolve. In any reason why it's not on the others as well?

Reply:
I'm pretty sure this comment is in all indexer's `resolve` and `reverse_resolve`, at least for me, maybe my VS Code is configured differently